pages/search/index.vue

```diff
@@ -61,6 +95,35 @@
 
     mounted() {
       this.$store.commit('search/enableCollectionFacet');
+      if (this.$route.query.query.length > 0) {
+        this.getSearchSuggestions(this.$route.query.query);
+      }
+    },
+
+    methods: {
+      async getSearchSuggestions(query) {
+
+        // Query in the user's language, removing duplicates
+        const languageParam = Array.from(new Set([this.$i18n.locale])).join(',');
+
+        const suggestions = await getEntitySuggestions(query, {
+          language: languageParam
+        }, {
+          recordValidation: this.enableSuggestionValidation
```

Comment:
Remove this as the record validation option is to be removed in another open PR.